cmd/blockchaincmd/deploy.go

```diff
@@ -182,6 +196,15 @@ so you can take your locally tested Blockchain and deploy it on Fuji or Mainnet.
 	posGroup := flags.AddProofOfStakeToCmd(cmd, &deployFlags.ProofOfStakeFlags)
 
 	cmd.SetHelpFunc(flags.WithGroupedHelp([]flags.GroupedFlags{networkGroup, bootstrapValidatorGroup, localMachineGroup, localNetworkGroup, nonSovGroup, icmGroup, posGroup, sigAggGroup}))
+
+	// enabling blockchain names, C-Chain and blockchain IDs
+	vmcChainFlags.SetEnabled(true, true, false, false, true)
+	vmcChainFlags.SetFlagNames("vmc-blockchain", "vmc-c-chain", "", "", "vmc-blokchain-id")
+	vmcChainFlags.AddToCmd(cmd, "deploy the Validator Manager contract into %s")
+	cmd.Flags().BoolVar(&vmcAtL1, vmcAtL1FlagName, false, "deploy the Validator Manager into the L1")
+	vmcPrivateKeyFlags.SetFlagNames("vmc-private-key", "vmc-key", "vmc-genesis-key")
+	vmcPrivateKeyFlags.AddToCmd(cmd, "as Validator Manager contract deployer")
```

Comment:
i think another name other than vmc can be better, not sure what

Reply:
lets create grouped flags for Validator Manager contract deployer, can be done in another pr

Reply:
for the moment on will keep vmc because alternative expansion validator-manager-contract seems pretty long to me. but accept suggestions